snapshot-labs/sx-monorepo
# fix: get ABI method not working

### Summary

Closes: https://github.com/snapshot-labs/workflow/issues/345

- getABI not working because of multiple reasons (wrong URL and no API KEY)
- Also can fetch proxy contract's ABI now

### How to test
- Go to http://127.0.0.1:8080/#/base:0x282d013BF31374D0046F04D6a7644d97751ed339/create/8uuja
- Under execution, click on "contract call"
- Use `0x833589fcd6edb6e08f4c7c32d4f71b54bda02913` and it should work

 Or 
- Go to http://127.0.0.1:8080/#/base:0x282d013BF31374D0046F04D6a7644d97751ed339/treasury/1/tokens
- use Uniswap to connect with wallet connect
- Try swapping USDC to ETH
